### PR TITLE
[TACHYON-552] bash script argument parsing fix

### PR DESCRIPTION
**Pull request by miasantreble**
https://tachyon.atlassian.net/browse/TACHYON-552

### CODE REVIEW

File: deploy/vagrant/create [hunk 1]
```diff
@@ -7,14 +7,14 @@ usage() {
  exit 1
 }
 
-if [[ "$#" != "2" ]]; then
+if [ "$#" -ne 2 ]; then
```
Comment (apc999) [inline deploy/vagrant/create]: "$#" or $# ?

File: deploy/vagrant/create [hunk 2]
```diff
@@ -7,14 +7,14 @@ usage() {
...
  usage
 fi
 
 TOTAL="$1"
 PROVIDER="$2"
 
-if [[ "$TOTAL" < "2" ]]; then
+if [ "$TOTAL" -lt 2 ]; then
```
Comment (apc999) [inline deploy/vagrant/create]: $TOTAL?

Comment (miasantreble) [inline deploy/vagrant/create]: I thought it's OK either way, no?
http://www.tldp.org/LDP/abs/html/comparison-ops.html
http://www.tldp.org/LDP/abs/html/quotingvar.html

Comment (gpang) [inline deploy/vagrant/create]: What happens when you run the script?

Comment (miasantreble) [inline deploy/vagrant/create]: oldnine@osboxes:~/git/tachyon/deploy/vagrant$ ./create 17 aws
ERROR: The environment variable AWS_ACCESS_KEY_ID must be set
oldnine@osboxes:~/git/tachyon/deploy/vagrant$ ./create 1 aws
number of machines should at least be 2
oldnine@osboxes:~/git/tachyon/deploy/vagrant$ ./create -1 aws
number of machines should at least be 2
oldnine@osboxes:~/git/tachyon/deploy/vagrant$ ./create -99 aws
number of machines should at least be 2

**Comment by gpang** [comment]
@uronce-cc

**Comment by gpang** [comment]
LGTM

**Comment by uronce-cc** [comment]
+1 LGTM

**Pull request merged**